proto/meta.proto

```diff
@@ -771,3 +771,20 @@ service EventLogService {
   rpc ListEventLog(ListEventLogRequest) returns (ListEventLogResponse);
   rpc AddEventLog(AddEventLogRequest) returns (AddEventLogResponse);
 }
+
+message ActorIds {
+  repeated uint32 ids = 1;
+}
+
+message FragmentIdToActorIdMap {
+  map<uint32, ActorIds> map = 1;
+}
```

Comment:
Can we nest them into `RelationIdInfos`?

Reply:
<img width="398" alt="Screenshot 2024-08-28 at 3 45 16 PM" src="https://github.com/user-attachments/assets/3a84a9b7-2582-47db-b77b-867d5026ab5d">

Seems nested map type not supported.

`Message, enum, or built-in type expected`.